.github/workflows/node.js.yml

```diff
@@ -18,3 +18,7 @@ jobs:
     - run: npm run build --if-present
     - name: Run tests
       run: npm test
```

Comment:
remove this step since is duplicated of the one you just added

Reply:
Thank you. Both lines?

Reply:
@holgiosalos  Can you see the second commit that I pushed?